ci-operator/step-registry/interop/oadp/execute/interop-oadp-execute-commands.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+SECRETS_DIR="/tmp/secrets"
+OADP_GH_PAT=$(cat ${SECRETS_DIR}/oadp/oadp-gh-pat)
+
+
+readonly OADP_GIT_DIR="${HOME}/cspi"
+readonly OADP_APPS_DIR="${HOME}/oadpApps"
+readonly PYCLIENT_DIR="${HOME}/pyclient"
+mkdir -p "${OADP_GIT_DIR}"
+mkdir -p /tmp/test-settings
+touch /tmp/test-settings/default_settings.json
+mkdir -p "${OADP_APPS_DIR}"
+mkdir -p "${PYCLIENT_DIR}"
+
+
+echo "Annotate oadp namespace"
+oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'
+
+
+echo "AWS info"
+cp "${CLUSTER_PROFILE_DIR}/.awscred" /tmp/test-settings/aws_creds
+echo "End of AWS info"
+
+
+
+echo "clone oadp-e2e-qe"
+git clone --branch master https://madunn:${OADP_GH_PAT}@github.com/CSPI-QE/oadp-e2e-qe "${OADP_GIT_DIR}"
```

Comment:
By using a PAT to git clone these repos are not allowing for this to be a very re-usable step in the step-registry.

This will require any team that needs to execute these tests to use their own PAT from a vault account. We can bring in the code from other private github repos by using the[ image promotion process](https://cspi-qe.gitbook.io/ocp-ci-onboarding/onboarding/onboarding_guide/scenario_development_guide#multiple-private-repo-environment-images) and having the code already in the environment when we build the container for the step. This may include creating multiple base_images from these repos then combining those base_images into a single image for this step if all repos need to be installed in a single container. This work needs to be further investigated.